app/src/main/java/gen/test/android/playlistmaker/PlayerActivity.kt

```diff
@@ -10,62 +13,143 @@ import androidx.constraintlayout.widget.Group
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.google.gson.Gson
-const val ROUNDED_CORNERS_PLAYER=8f
+
+const val ROUNDED_CORNERS_PLAYER = 8f
 const val KEY_PLAYER_ACTIVITY = "KEY_PLAYER_ACTIVITY"
+
 class PlayerActivity : AppCompatActivity() {
 
+    companion object {
+        private const val STATE_DEFAULT = 0
+        private const val STATE_PREPARED = 1
+        private const val STATE_PLAYING = 2
+        private const val STATE_PAUSED = 3
+        private const val UPDATE_UI = 300L
+    }
+
+    private var playerState = STATE_DEFAULT
+    private lateinit var playBtn: ImageButton
+    private lateinit var timePlayTV: TextView
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Объект плеера не пересоздаётся в рамках экрана